docs/intro/intro_cqm.rst

```diff
@@ -13,7 +13,47 @@ typically formulate the constraints as penalties. Constrained models such as
 :class:`ConstrainedQuadraticModel` can support constraints by encoding both an
 objective and its set of constraints, as models or in symbolic form.
 
-CQM
-===
+Supported Models
+================
 
-TODO
+dimod provides a :term:`constrained quadratic model` (CQM) class that encodes
+a quadratic objective and one or more quadratic equality and inequality constraints.
```

Comment:
technically there can be no constraints.

Reply:
Added "possibly"